### ISSUE TITLE
Where does the data for translation training come from?

### ISSUE THREAD

**Issue by StoyanVenDimitrov** [ufal/neuralmonkey#558]
Hi, 

I need to be able to tell where the data sets on http://ufallab.ms.mff.cuni.cz/~helcl/neuralmonkey-example-data/translation/ come from. But I didn´t found the information anywhere. 

Best,
Stoyan

**Comment by jindrahelcl** [comment]
Hi,

I am not sure, but I think the data are a part of the EN-DE WMT data. Note
that this dataset is not complete and should not be used for any real
experiments..

J

2017-10-21 15:36 GMT+02:00 StoyanVenDimitrov <notifications@github.com>:

**Comment by StoyanVenDimitrov** [comment]
Ok, then I'll take the WMT14 dataset, it will also be comparable with the results of your paper. But the link  http://www.statmt.org/wmt14/translation-task.html doesn´t work. Any idea where I can get the data from?

Best, 
Stoyan

**Comment by StoyanVenDimitrov** [comment]
Hi,

I see here  http://www.statmt.org/wmt14/translation-task.html  multiple news corpora. Which one do you used exactly to test the NM?

Best,
Stoyan

**Comment by jindrahelcl** [comment]
Hi,
for the Neural Monkey paper, we used the newstest2014 FR->EN dataset (
https://ufal.mff.cuni.cz/pbml/107/art-helcl-libovicky.pdf)

J


2017-10-23 10:45 GMT+02:00 StoyanVenDimitrov <notifications@github.com>:

**Comment by StoyanVenDimitrov** [comment]
Do you mean exactly the 2.3 GB 10^9 French-English corpus

**Comment by jindrahelcl** [comment]
No I mean the TEST set for the 2014 task. It should be few thousand
sentences at most, definitely under 5 MB.

2017-10-23 10:53 GMT+02:00 StoyanVenDimitrov <notifications@github.com>:

**Comment by StoyanVenDimitrov** [comment]
All right, thank you. And any idea which set you used for training:  Europarl, UN Corpus...?

**Comment by jindrahelcl** [comment]
For the numbers in the paper, I think we used all the data available from
the WMT website (including europarl, UN, news commentary, and others)

2017-10-23 13:29 GMT+02:00 StoyanVenDimitrov <notifications@github.com>:

**Comment by StoyanVenDimitrov** [comment]
Thanks, I guess I'll take only one of them... in order to finish at least a whole epoch :D

**Comment by StoyanVenDimitrov** [comment]
Hi, how did you converted the test set from .sgm? I have a lot of troubles with running the `wrap_xml.perl` from Moses and am looking for an alternative.

**Comment by jindrahelcl** [comment]
I think that script is used when you want to convert text to sgm, not the other way around. From SGM, it suffices that you strip the tags and leave the text.

Here's a ranfom python function that does it:
https://github.com/tensorflow/tensor2tensor/blob/9bdc801e23586321e04e743fc61dd66b10adbc1d/tensor2tensor/data_generators/translate.py#L172

**Comment by StoyanVenDimitrov** [comment]
You are right. Now I have everything. Thanks!
